tests/udf/test_simple_udf.py

```diff
@@ -53,3 +55,26 @@ def my_frame_fn(frame, pixelsum):
     assert 'pixelsum' in res
     print(data.shape, res['pixelsum'].data.shape)
     assert np.allclose(res['pixelsum'].data, np.sum(data, axis=(1, 2)))
+
+
+def test_minibatch(lt_ctx):
```

Comment:
You can move this function to its own file, for example `tests/udf/test_stddev.py`. It should also have a more descriptive name (it is no longer testing `minibatch`, right?)

In addition to this test with random data, a simple test with "hand-made" data would be useful, where someone could calculate the results by hand and can easily convince themselves that the test case and expected results is correct.